examples/archive.html

```diff
@@ -95,9 +95,6 @@ <h3 id="offcanvasRightLabel">Images</h3>
     form.addEventListener('submit', (event) => {
       event.preventDefault();
       const url = input.value.replace('details', 'metadata');
-      let splitUrl = url.split('/');
```

Comment:
First, just noting this seems like an unrelated change -- can you try reverting it with:

```sh
cd examples
wget https://raw.githubusercontent.com/publiclab/Leaflet.DistortableImage/main/examples/archive.html
cd ../
git add dist
git commit -m 'removed unrelated changes'
git push
```